pages/linux/lslocks.md

```diff
@@ -0,0 +1,23 @@
+# lslocks
+
+> List local system locks.
+
+- List all local system locks:
+
+`lslocks`
+
+- List locks with defined column headers:
+
+`lslocks --output PID,COMMAND,PATH`
```

Comment:
Should PID,COMMAND,PATH be within tokens (`{{}}`) ?

Reply:
You're right, it should.